connectors/src/connectors/slack/temporal/worker.ts

```diff
@@ -12,6 +15,13 @@ export async function runSlackWorker() {
     maxConcurrentActivityTaskExecutions: 1,
     connection,
     namespace,
+    interceptors: {
```

Comment:
If we agree on this design, all workers must define this interceptor. We can also provide a help function for that.